### PR TITLE
RHBPMS-1591 - Fix missing i18n for perspective title

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by manstis** [comment]
+1 to merge (when green).

File: uberfire-extensions/uberfire-runtime-plugins/uberfire-runtime-plugins-client/src/main/resources/org/uberfire/ext/plugin/client/resources/i18n/CommonConstants.properties [hunk 1]
```diff
@@ -68,4 +68,5 @@ ScreenComponent=Screen Component
 ScreenConfiguration=Screen Configuration
 PlaceName=Place Name
 EmptyTitleText=Create a new Perspective
-EmptySubTitleText=Drag a Component here to start building a new perspective.
\ No newline at end of file
+EmptySubTitleText=Drag a Component here to start building a new perspective.
```
Comment (ederign) [inline uberfire-extensions/uberfire-runtime-plugins/uberfire-runtime-plugins-client/src/main/resources/org/uberfire/ext/plugin/client/resources/i18n/CommonConstants.properties]: maybe here is the conflict with https://github.com/uberfire/uberfire/pull/479/files

**Comment by ederign** [comment]
@cristianonicolai could you please fix the conflicts?

**Comment by cristianonicolai** [comment]
Closing this one to avoid conflicts with the work that @paulovmr is doing here: https://github.com/uberfire/uberfire-extensions/pull/290

**Pull request closed without merging**